frontend/src/components/pages/TeamDashboard/index.jsx

```diff
@@ -59,14 +64,20 @@ function DashboardUnconnected({
       userIds: team ? team.members.map((member) => member.userId) : [],
     }),
   };
+  if (activityLogDayCounts.length === 0)
+    getActivityLogCountsByDayForUsers({
+      activityLogDayCounts,
+      eventTypes,
+      userIds: team ? team.members.map((member) => member.userId) : [],
+    });
```

Comment:
I don't think this code does anything...

getActivityLogCountsByDayForUsers returns an object. It doesn't change anything. You are computing a value and then doing nothing with it.  What is your intention here?